lib/main.js

```diff
@@ -1,38 +1,54 @@
-// stake is in pence
-// £10.00 becomes 1000
+import combinations from './combinations';
 
-// odds are a decimal
-// "8/1" becomes 9.0
+function bet(stake, odds) {
+  return (stake * (odds - 1)) + stake;
+}
 
-// terms are decimal
-// "1/5" becomes 0.2
+function single({
+  stake, odds, terms, ew = false, placeStake = 0
+}) {
+  const newPlaceStake = ew && placeStake === 0 ? stake : placeStake;
 
-// single(stake, odds, terms) {
+  const placeReturn = ew ? bet(newPlaceStake, ((odds - 1) * terms) + 1) : 0;
 
-  module.exports = (function() {
-    function bet(stake, odds) {
-      return (stake * (odds - 1)) + stake;
-    }
+  const winReturn = bet(stake, odds);
 
-    function single(stake, odds, terms, placeStake = 0) {
-      const newPlaceStake = terms && placeStake === 0 ? stake : placeStake;
+  return {
+    win: winReturn,
+    place: placeReturn,
+    total: (winReturn + placeReturn)
+  };
+}
 
-      const placeReturn = terms ? bet(newPlaceStake, ((odds - 1) * terms) + 1) : 0;
+function accumulator(selections, stake, ew, idx = 0, place = 0) {
+  const sel = selections[idx];
 
-      const winReturn = bet(stake, odds);
+  const res = single({
+    stake, odds: sel.odds, terms: sel.terms, ew, placeStake: place
+  });
 
-      return { win: winReturn, place: placeReturn, total: (winReturn + placeReturn) };
-    }
+  if (idx + 1 === selections.length) {
+    return res.total;
+  }
 
-    function multiple(selections, stake, idx = 0, place) {
-      const sel = selections[idx];
+  return accumulator(selections, res.win, ew, idx + 1, res.place || 0);
+}
 
-      const res = single(stake, sel.odds, sel.terms, place);
+function multiple(selections, stake, ew = false, fullCover = false) {
+  const count = selections.length;
+  const start = fullCover ? 1 : 2;
+  let allCombos = [];
 
-      if (idx + 1 === selections.length) return res.total;
+  for (let i = start; i <= count; i += 1) {
+    allCombos = allCombos.concat(combinations(selections, i));
+  }
 
-      return multiple(selections, res.win, idx + 1, res.place || 0);
-    };
+  return allCombos.reduce((acc, selection) => acc + accumulator(selection, stake, ew), 0);
```

Comment:
to make this bit a bit nicer to read, maybe do a similar declaration of allCombos = THIS then you can just have a nice line at the end with return allCombos.